templates/content/faq.html.twig.save

```diff
@@ -0,0 +1,456 @@
+{% extends 'base.html.twig' %}
```

Comment:
Always check your PR to ensure you didn't push unwanted diff :slightly_smiling_face:

Reply:
Oops, my bad